homeassistant/components/bmw_connected_drive/__init__.py

```diff
@@ -20,13 +20,15 @@
 
 DOMAIN = 'bmw_connected_drive'
 CONF_REGION = 'region'
+CONF_ENABLE_SERVICES = 'services'
```

Comment:
Can you rename the option `read_only` and default to `False`

Reply:
do you mean CONF_READ_ONLY = 'services' ? This wouldnt make it consistent, because there is always a negation needed. 
I could do CONF_READ_ONLY = 'services_disabled' and set the default to false ?